snippets/ztp-patch-argocd-hub-cluster.adoc

```diff
@@ -1,5 +1,71 @@
 :_mod-docs-content-type: SNIPPET
-. To install the {ztp} plugin, patch the ArgoCD instance in the hub cluster by using the patch file that you previously extracted into the `out/argocd/deployment/` directory.
+. To install the {ztp} plugin, patch the ArgoCD instance in the hub cluster with the relevant multicluster engine (MCE) subscription image.
+Customize the patch file that you previously extracted into the `out/argocd/deployment/` directory for your environment.
+
+.. Select the `multicluster-operators-subscription` image that matches your {rh-rhacm} version.
++
+--
+.`multicluster-operators-subscription` image versions
+[cols="1,1,1,1,3", options="header"]
+|====
+|{product-title} version
+|{rh-rhacm} version
+|MCE version
+|MCE RHEL version
+|MCE image
+
+|4.14, 4.15, 4.16
+|2.8, 2.9
+|2.8, 2.9
+|RHEL 8
+|`registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel8:v2.8`
+
+`registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel8:v2.9`
+
+|4.14, 4.15, 4.16
+|2.10
+|2.10
+|RHEL 9
+|`registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel8:v2.10`
```

Comment:
base image version is rhel9 hence "-rhel8" needs to be changed to be "-rhel9". here is the reference from original dev PR:- https://github.com/openshift-kni/cnf-features-deploy/pull/1890/files#diff-a5b2fbb23a415e4c707c5d404ed7656ef83e319e6725d52f86be1c9d0e927021R52

Reply:
Well spotted thanks